excel/beginner/charts/why-are-charts-useful.md

```diff
@@ -0,0 +1,42 @@
+---
+author: Stefan-Stojanovic
+
+levels:
+  - beginner
+
+aspects:
+  - new
+  - workout
+
+type: normal
+
+category: how to
+
+---
+
+# Why are charts useful?
+
+---
+## Content
+
+A good example of why charts are useful is because they can be used to detect patterns and understand results in a more clear way.

```

Comment:
Maybe we could also add the reference to the famous quote "A picture tells a 1000 words"